JavaScript/handwriting/public/js/main.js

```diff
@@ -0,0 +1,248 @@
+(function() {
+  var addClick, canvas, classify, clickDrag, clickX, clickY, client, context, dragging, drawPath, erase, getBoundingBox, mahoutClassify, miniCanvas, miniContext, paint, redraw, strokeWidth, toGreyscale, wekaClassify;
+
+  canvas = null;
+
+  context = null;
+
+  miniCanvas = null;
+
+  miniContext = null;
+
+  strokeWidth = 16;
+
+  dragging = false;
+
+  paint = false;
+
+  clickX = [];
+
+  clickY = [];
+
+  clickDrag = [];
+
+  // this API Key will only work on Algorithmia's website; get your own key at https://algorithmia.com/user#credentials
+  client = Algorithmia.client('simYGhuyUpyFFAIRynUx1CKzWdI1');
```

Comment:
:bulb: Let's add a quick note here re/ who owns this key and it's relevant permissions.

Reply:
Added to demo key tracking sheet!